src/components/Footer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+import { InnerWrapper } from '../library/GlobalStyles'
+
```

Comment:
I like how neat the whole footer component you made here and I can read clearly what each this is supposed to be. Good work, Linda!

My only suggestion is to change the <div> into maybe <p> so it is clearer what the content of that section is.